internal/configuration/env_config.go

```diff
@@ -13,7 +13,7 @@ import (
 const (
 	defaultIdleLongBuild           = 3
 	defaultIdleAfter               = 45
-	defaultMaxRetries              = 3
+	defaultMaxRetries              = 10
```

Comment:
do you know if this is what gets used in the actual deployment? I am sure these values can be overridden though env vars or  as options to `idler` binary. So could we ensure that only the default value is used in prod and if that is not the case, make sure the deployment config is updated to reflect the max retry value of 10?

Reply:
I did not find any occurrence of `JC_MAX_RETRIES` in fabric8-tenant-jenkins https://github.com/fabric8-services/fabric8-tenant-jenkins/search?q=JC_MAX_RETRIES, so they're not overriding this value I believe.

Reply:
@containscafeine sorry, I didn't follow. Why are we looking for the env var in tenant-jenkins repo? Does idler deployment rely on that?

doesn't it require fixing https://github.com/fabric8-services/fabric8-jenkins-idler/blob/dd784b50c2b3634c562ce5b6af1181c2f44bd97c/openshift/jenkins-idler.app.yaml#L38 instead?

Reply:
@sthaha thanks for pointing that out, changed the value there